mainlyetcetera/rotten_tomatillos
# Tests/cypress tests

## Is this a fix or a feature?
- feature
## What is the change/fix?
- adds tests for the entire app using `cypress`
## Where should the reviewer start?
- `cypress/integration/App-spec.js`
## How should this be tested?
- pull down branch (not to main initially)
- run `npm run cypress`
- observe all tests passing
## Screenshots(if appropriate)
![image](https://user-images.githubusercontent.com/70294115/107888885-9230fd80-6ecc-11eb-857f-eb1c3f3659f8.png)
## This PR is related to which issues:
- none
